remappings.txt

```diff
@@ -0,0 +1 @@
+openzeppelin/=contracts/
```

Comment:
Oh, we also have to discuss if we want `openzeppelin`, `@openzeppelin`, or something else. Right now I'm fine with this remapping.

Reply:
We haven't added a remappings file before, so we can have this discussion later outside of this PR. For now all good, thanks @pcaversaccio

Reply:
**__**